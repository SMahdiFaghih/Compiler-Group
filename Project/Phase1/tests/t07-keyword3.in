VOID
INT
DOUBLE
BOOL
STRING
CLASS
INTERFACE
NULL
THIS
EXTENDS
IMPLEMENTS
FOR
WHILE
IF
ELSE
RETURN
BREAK
NEW
NEWARRAY
PRINT
READINTEGER
READLINE
