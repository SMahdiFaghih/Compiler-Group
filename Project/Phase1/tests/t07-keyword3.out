T_ID VOID
T_ID INT
T_ID DOUBLE
T_ID BOOL
T_ID STRING
T_ID CLASS
T_ID INTERFACE
T_ID NULL
T_ID THIS
T_ID EXTENDS
T_ID IMPLEMENTS
T_ID FOR
T_ID WHILE
T_ID IF
T_ID ELSE
T_ID RETURN
T_ID BREAK
T_ID NEW
T_ID NEWARRAY
T_ID PRINT
T_ID READINTEGER
T_ID READLINE
